Makefile

```diff
@@ -394,6 +395,11 @@ openstack_crds: ## installs all openstack CRDs. Useful for infrastructure dev
 	for X in $$(file ${OUT}/openstack_crds/* | grep gzip | cut -f 1 -d ':'); do tar xvf $$X -C ${OUT}/openstack_crds/; done
 	for X in $$(grep -l CustomResourceDefinition ${OUT}/openstack_crds/manifests/*); do oc apply -f $$X; done
```

Comment:
@cescgina I like that we are sharing the function, but now wonder if the fact that we extract things into the same openstack_crds directory might eventually cause a side effect. We could perform a 'basename' on the URL provided. Or just default to 'openstack_crds' but have an separate ENV variable that could override it for the 'storage' version? What do you think? We could also do this in a follow up I think.